drivers/hmis/app/models/hmis/hud/validators/exit_validator.rb

```diff
@@ -40,6 +44,14 @@ def self.validate_exit_date(exit, household_members: nil, options: {})
     errors.add :exit_date, :out_of_range, message: before_entry_message(entry_date), **options if entry_date.present? && entry_date > exit_date
     errors.add :exit_date, :information, severity: :warning, message: over_thirty_days_ago_message, **options if exit_date < (Date.today - 30.days)
     errors.add :exit_date, :out_of_range, message: before_dob_message, **options if dob.present? && dob > exit_date
+
+    conflict_scope = enrollment.project
+      .enrollments_including_wip
+      .where(personal_id: enrollment.personal_id, data_source_id: enrollment.data_source_id)
+      .with_conflicting_dates(entry_date...exit_date)
+
+    conflict_scope = conflict_scope.where.not(id: enrollment.id) if enrollment.persisted?
```

Comment:
This seems odd to me that you'd prevent closing an enrollment that overlaps another enrollment.  I guess it's good to call out, but I'm not sure how you'd fix it.

Reply:
In some cases it could be fixed by moving the exit date earlier. Maybe lets update this one to `severity: :warning` so that user can bypass it.

Reply:
I think that'd be safer